packages/shorebird_cli/lib/src/commands/patch/patch_ios_command.dart

```diff
@@ -162,14 +163,15 @@ If this option is not provided, the version number will be determined from the p
 
     const arch = 'aarch64';
     const releasePlatform = ReleasePlatform.ios;
-    final flavor = results['flavor'] as String?;
+    final flavor = results.findOption('flavor', argParser: argParser);
+    final target = results.findOption('target', argParser: argParser);
 
     final shorebirdYaml = shorebirdEnv.getShorebirdYaml()!;
     final appId = shorebirdYaml.getAppId(flavor: flavor);
     final app = await codePushClientWrapper.getApp(appId: appId);
 
     try {
-      await _buildPatch();
+      await _buildPatch(flavor: flavor, target: target);
```

Comment:
Why are these now passed to buildPatch and they were not before?

Reply:
This makes it so we're only looking up the arg once. Minor optimization, but seemed better 🤷